test/unit/lib/plugins/aws/package/compile/events/schedule.test.js

```diff
@@ -223,6 +265,49 @@ describe('test/unit/lib/plugins/aws/package/compile/events/schedule.test.js', ()
       InputTemplate: '{"time": <eventTime>, "key": "value"}',
       InputPathsMap: { eventTime: '$.time' },
     });
+    expect(scheduleCfResources[8].Properties.Target.InputTransformer).to.be.undefined;
+    expect(scheduleCfResources[9].Properties.Target.InputTransformer).to.be.undefined;
+  });
+
+  it('should throw when passing "inputPath" or "inputTransformer" to method:schedule resources', async () => {
+    const events = [
+      {
+        schedule: {
+          rate: 'rate(15 minutes)',
+          method: 'scheduler',
+          inputPath: '$.stageVariables',
+        },
+      },
+      {
+        schedule: {
+          rate: 'rate(15 minutes)',
+          method: 'scheduler',
+          inputTransformer: {
+            inputPathsMap: { eventTime: '$.time' },
+            inputTemplate: '{"time": <eventTime>, "key": "value"}',
+          },
+        },
+      },
+    ];
+
+    await expect(run([events[0]])).to.be.eventually.rejectedWith(
+      ServerlessError,
+      "Configuration error at 'functions.test.events.0.schedule.method': must be equal to one of the allowed values [eventBus]\n\nLearn more about configuration validation here: http://slss.io/configuration-validation"
```

Comment:
Let's validate against error`code`  and not `message` (as we do in other error handling tests)